### ISSUE TITLE
Unexpected key 'moduleId' found

### ISSUE THREAD

**Issue by kobezzza** [rollup/rollup#379]
```
Unexpected key 'moduleId' found, expected one of: entry, dest, plugins, external, onwarn, indent, format, moduleName, sourceMap, intro, outro, banner, footer, globals, transform, load, resolveId, resolveExternal
```

**Comment by TrySound** [comment]
@kobezzza Show config code, input code

**Comment by kobezzza** [comment]
```js
'use strict';

export function foo() {

}
```

```bash
rollup ./src/index.js --id Snakeskin --name Snakeskin --format umd
```

```
Unexpected key 'moduleId' found, expected one of: entry, dest, plugins, external, onwarn, indent, format, moduleName, sourceMap, intro, outro, banner, footer, globals, transform, load, resolveId, resolveExternal
Error: Unexpected key 'moduleId' found, expected one of: entry, dest, plugins, external, onwarn, indent, format, moduleName, sourceMap, intro, outro, banner, footer, globals, transform, load, resolveId, resolveExternal
    at validateKeys (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/www/ROLLUP/rollup/src/utils/validateKeys.js:12:11)
    at Object.rollup (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/www/ROLLUP/rollup/src/rollup.js:41:16)
    at bundle (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/bin/runRollup.js:114:16)
    at execute (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/bin/runRollup.js:103:3)
    at module.exports (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/bin/runRollup.js:58:3)
    at Object.<anonymous> (/Users/andrejkobec/.nvm/versions/node/v5.3.0/lib/node_modules/rollup/bin/rollup:36:26)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
```

**Comment by Victorystick** [comment]
`moduleId` is a valid key, but it isn't included in the check added by 934d1e1133c74818421a953398c166f66e2583ef.

**Comment by kobezzza** [comment]
With Rollup 0.20 all fine.

**Comment by Rich-Harris** [comment]
Fix released as 0.22.1.
